### PR TITLE
BL-365 [BACK END] patch/instructor_newsfeed/sort_posts_newest_to_oldest

### PR DESCRIPTION
**Pull request by anastasia-lapteva**
## Description

Jira Ticket # **BL-365** Patch: InstructorNewsFeed currently list posts from oldest to newest.

- changed getNewsfeed in newsfeedModel so that it is ordered by posted_at in DESC order
- changed newsfeed migration so that posted_at is notNullable
- added 1 more seed in 013_newsfeed

Co-Author: **Vasilii Garanin**

## Fixes

InstructorNewsFeed now displays the most recent posts at the top of the web browser.

## Loom Video

https://www.loom.com/share/78515bca4dd14a8b9e9b60080e0d1d92

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have removed unnecessary comments/console logs from my code
- [X] I have made corresponding changes to the documentation if necessary (optional)
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] No duplicate code left within changed files
- [X] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by Rlauckern — DISMISSED** [review]

**Review by Roadlyfe — APPROVED** [review]
I'm a little confused by the check failures on data migrations so I'm going to look that up. Great work!

**Review by reneetro — DISMISSED** [review]
I'm not quite familiar with the "check failures" on the migrations, I will have to do some research. Everything else looked clean!

**Comment by anastasia-lapteva** [comment]
Thanks for the feedback, everyone. It looks like checks are failing because eslint is not happy with my spacing, and `npm run format` is not working. I messaged with Ash Tilawat and he is looking into this.

**Review by Roadlyfe — DISMISSED** [review]
nice work!

**Review by AnthonyLayne — DISMISSED** [review]
Investigated the check failures a bit, it looks like it may not like the double quotes on line 22 for the description in the seed file. From what I could find it looks like you can't use double quotes in the database. Could be wrong though. If you click on the note icon with the checkmark on the top right of the check failure, you can see more of the error.

**Comment by anastasia-lapteva** [comment]
@AnthonyLayne thanks for your help! I initially went with double quotes to avoid using the \ backslash in the description text. But based on your comment I swapped back to single quotes and added the backslash before apostrophes. I also added in eslint-disable for good measure and now checks are passing :-)

**Review by AnthonyLayne — DISMISSED** [review]
Your code looks good. Thank you for fixing the errors and fixing render order of the newsfeed once an article has been edited. Great Job!

**Review by Cassius-Cassity — DISMISSED** [review]
Pardon my ignorance but why did so many lines on package-lock.json have so many deletions?

**Review by Irving-Delgad0 — DISMISSED** [review]
Yeah im gonna second the package.json question. seems like there are a lot of changes. seems unusual but i could be wrong

**Review by dshipuk — DISMISSED** [review]
As somebody mentioned the package-lock.json has a lot of deletions seems like an accident occurred somebody should review this before merging

**Review by techlando — DISMISSED** [review]
Not sure why so much deleted code
But the written code is good. Great job adding data tables and schema

**Review by Rlauckern — DISMISSED** [review]
Good tables and schema! Looks like a lot needed to change and written well.

**Review by gclancy121 — DISMISSED** [review]
looks good to me

**Review by Roadlyfe — DISMISSED** [review]
Looks great! No obvious bugs that  I can see.

File: data/migrations/20211103214447_newsfeed.js
Comment (ryan-hamblin) [inline data/migrations/20211103214447_newsfeed.js:1]: Let's not disable disable our linter.

Comment (ryan-hamblin) [reply data/migrations/20211103214447_newsfeed.js:1]: Please run the `npm lint-fix` command to format your documents properly in lieu of disabling the linter.

File: data/seeds/013_newsfeed.js
Comment (ryan-hamblin) [inline data/seeds/013_newsfeed.js:1]: please don't disable the linter.

**Review by ryan-hamblin — COMMENTED** [review]
Straightforward work. Please don't disable our linter. It's there to project everyone. Instead be sure your code formats to the proper style outlined in your `lint/prettier rc` files

**Pull request merged**
